tensorflow/inference/docker/2.0.2/py3/Dockerfile.gpu

```diff
@@ -0,0 +1,130 @@
+FROM nvidia/cuda:10.0-base-ubuntu18.04
+
+LABEL maintainer="Amazon AI"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG PYTHON=python3
```

Comment:
Just to double check, python3 for ubuntu18 defaults to 3.6 right?

Reply:
yes